packages/jaeger-ui-components/src/TracePageHeader/SpanGraph/Scrubber.tsx

```diff
@@ -85,30 +85,41 @@ export default function Scrubber({ isDragging, onMouseDown, onMouseEnter, onMous
   const styles = useStyles2(getStyles);
   const className = cx({ [styles.ScrubberDragging]: isDragging });
   return (
-    <g className={className}>
-      <g
-        className={styles.ScrubberHandles}
-        onMouseDown={onMouseDown}
-        onMouseEnter={onMouseEnter}
-        onMouseLeave={onMouseLeave}
-      >
-        {/* handleExpansion is only visible when `isDragging` is true */}
-        <rect
-          x={xPercent}
-          className={styles.ScrubberHandleExpansion}
-          style={{ transform: `translate(-4.5px)` }}
-          width="9"
-          height="20"
-        />
-        <rect
-          x={xPercent}
-          className={styles.ScrubberHandle}
-          style={{ transform: `translate(-1.5px)` }}
-          width="3"
-          height="20"
+    <svg>
```

Comment:
Why this added `<svg>` element? This component is already embedded in another `<svg>` in the parent so it does not seem like it should be here as it creates `<svg><svg></svg></svg>`.